autoload/neomake/makers/ft/rst.vim

```diff
@@ -23,3 +23,19 @@ function! neomake#makers#ft#rst#rstcheck()
             \ '%E%f:%l: (SEVERE/4) %m',
         \ }
 endfunction
+
+" TODO: determine proper path of current .rst file upwards in sear for conf.py file, use that instead of "docs"
+" TODO: ask noomake for a temporary folder instead of /tmp/neomake-sphinx
+" TODO: split arguments and make them configurable
```

Comment:
ideally i'd like to append or prepend options to make them compliment _sphinx_args, how can i do that? My VIM scripting knowledge is close to zero.